__tests__/app.test.js

```diff
@@ -92,6 +94,31 @@ describe('appTests', () => {
                     expect(review).toHaveProperty('created_at', expect.any(String))
                 })
         })
+        test('GET: 200 /api/reviews/:review_id/comments', () => {
+            return request(app)
+                .get('/api/reviews/3/comments')
+                .expect(200)
+                .then(({body})=>{
+                    const comments = body.reviewComments;
+                    expect(Array.isArray(comments)).toBe(true)
+                    expect(Object.prototype.toString.call(comments[0])).toBe('[object Object]');
+    
+                    comments.forEach((comment)=>{
+                        expect(Object.keys(comment).length).toBe(6)
+                        expect(comment).toHaveProperty('comment_id', expect.any(Number))
+                        expect(comment).toHaveProperty('votes', expect.any(Number))
+                        expect(comment).toHaveProperty('created_at', expect.any(String))
+                        expect(comment).toHaveProperty('author', expect.any(String))
+                        expect(comment).toHaveProperty('body', expect.any(String))
+                        expect(comment).toHaveProperty('review_id', expect.any(Number))
+                    })
+                    // should return in descending order (most recent first)
+                    for (let i = 1; i < comments.length; i++) {
+                        expect(Date.parse(comments[i-1].created_at) > Date.parse(comments[i].created_at)).toBe(true)
+                    }
+                })
+        })
+        
```

Comment:
This is a great test but we do have some more things to consider here such as;
- what happens if i request an article that has no comments but the article does exist?
- what happens if i request the comments for an article that doesn't exist?
- what happens if i request the comments for an article id that is not a number `/api/reviews/banana/comments` ?

Add these tests and then send a pr through tomorrow :)